src/lang/en-US.json

```diff
@@ -561,6 +561,7 @@
   "story_item_purchase": "{player} purchased a {item} at {time}",
   "story_predicted_victory": "{players} predicted {team} would win",
   "story_predicted_victory_empty": "No one",
+  "story_networth_diff": "{percent}% / {gold} Diff",
```

Comment:
I did it this way because different languages might prefer something other than '/' and also the order/placement of the 'Diff' could be different in other languages as well.